class BadCustomerInfo(Exception):

    def __init__(self):
        pass
